recognition/include/pcl/recognition/surface_normal_modality.h

```diff
@@ -45,6 +45,14 @@
 #include <pcl/point_types.h>
 #include <pcl/features/linear_least_squares_normal.h>
 
+#include <algorithm>
+#include <cmath>
+#include <cstddef>
+// #include <iostream>
```

Comment:
Any reason to keep the comment around?

Reply:
Commented lines like https://github.com/PointCloudLibrary/pcl/blob/9427da1fac4088e61e037cd3e0d4dedbdfebcd0b/recognition/include/pcl/recognition/surface_normal_modality.h#L1122